src/Appwrite/Utopia/Fetch/Client.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Appwrite\Utopia\Fetch;
+
+use Utopia\Fetch\Client as UtopiaClient;
+use Utopia\Fetch\Response;
+
+class Client extends UtopiaClient
```

Comment:
Can we use utopia/fetch instead? I think @Meldiron created a new improved version of it.

Reply:
I'll see if we can set base URL etc

Reply:
Isn't this possible in utopia/fetch?? Maybe we should add it? Not a blocker, but do ask @Meldiron about it.